spectrometer.cabal

```diff
@@ -507,6 +509,7 @@ test-suite unit-tests
     , hspec-expectations-pretty-diff  ^>=0.7.2.5
     , hspec-hedgehog                  ^>=0.0.1.2
     , hspec-megaparsec                ^>=2.2
+    , HUnit                           ^>=1.6.0
```

Comment:
I need to get at `assertFailure :: String -> IO a` to put in the API mocks.  The `expectationFailure :: String -> IO ()` is locked to returning unit and so it makes for nasty lint warnings if I want it to fit in something like `m a`